geoprocessing/src/migrations/geoprocessing/1614256630000-AddCountriesView.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddCountriesView1614256630000
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+CREATE UNIQUE INDEX only_countries_by_gid ON admin_regions(gid_0) WHERE (gid_0 IS NOT NULL AND gid_1 IS NULL AND gid_2 IS NULL);
```

Comment:
i suggest to drop this indexes from the migration, as the table is a partitioned one and after the patch mentioned in #74 this is no longer needed

Reply:
ok, done, thanks!